src/greaseweazle/image/scp.py

```diff
@@ -50,6 +51,16 @@
 }
 
 
+class SCPHeaderFlags(IntFlag):
+    INDEXED = auto()        # if set, the image used the index mark to queue tracks
```

Comment:
I don't agree with using `auto()` here. Value assignments are not guaranteed by the library API. Spell out the assignments. `INDEXED = 1` and so on.

Reply:
Ok, maybe the assigned values are guaranteed by the API. But I still don't agree with use of `auto()` when specific values are required. So switch to `1, 2, 4, ...` or `1<<0, 1<<1, 1<<2, ...`

Reply:
Yeap, I agree. Better to be explicit on the values. Fixed :+1: